R/incidence_to_R.R

```diff
@@ -9,64 +9,75 @@
 #'
 #' @seealso [def_dist_generation_interval()]
 #' @seealso [EpiEstim::make_config()]
+#'
 incidence_to_R <- function(
     incidence,
     generation.interval,
     prm.R
 ){
   # === prep inputs ====
 
-  # make config
+  # Prepare the configuration for EpiEstim's function
   incid  <- incidence$I
-  method <- "non_parametric_si"
+  tmp    <- prm.R[['method.EpiEstim']]
```

Comment:
@davidchampredon calling `method.EpiEstim` from `prm.R` implies that the format of `prm.R` had changed, which is a breaking change. this change of format isn't reflected in the documentation for `prm.R` (see [`man-roxygen/param-prm.R.R`](https://github.com/phac-nml-phrsd/ern/blob/dev-dc/man-roxygen/param-prm.R.R)).

(i discovered when merging `dc-dev` into my current development branch `cl-daily-input` in anticipation of `dc-dev` being merged into `main` first.)

what should the format of the `prm.R` list be? is this code assuming
```
list(
iter,
CI,
window,
config.EpiEstim,
method.EpiEstim
)
```
do we need to change the format of `prm.R` or should `method.EpiEstim` be grabbed internally from `prm.R$config.EpiEstim`?

do we want to let the user change the method of `EpiEstim`? i believe the rest of our code (re-sampling GI, etc) relies on `method = "non_parametric_si"`? i wrote warnings to this effect (if a user specifies their own method in the config, they're warned that `ern` ignores that because it needs `non_parametric_si`. 

i believe this is all related to the issue of bogus `method` argument in some `EpiEstim` function, but i forget the details...

@wyusuf068 should also be aware of this as we share the `prm.R` list across the ww and cl sides of the codebase.

Reply:
Thanks for flagging this @papsti . This is fixed in 6e48d53a92b15c456203e8647ff0c03e2cb0d832